### DISCUSSION TITLE
Dahua/Loryta cam video encoding and picture/exposure settings

### DISCUSSION THREAD

**Discussion by dlong500** [blakeblackshear/frigate#5846] [General]
I've purchased one of the exact recommended models (IPC-T5442TM-AS), and there are a lot of camera settings with which to work.  I'm curious to see what others are using for the various settings.  Of course I know that each environment may be different and need dialing in, but I'd still love to see some baseline camera configurations (camera config, not frigate config).

I'm specifically interested in the picture/exposure/backlight/WB settings as well as the video encoding settings (for both H.264 and H.265) that people may be using.  I'm having a bit of difficulty trying to dial in settings to achieve the best picture with the least amount of noise/blockiness as well as eliminating stutter.  The I-frame generation in particular seems to create a stutter in the stored video (I can see that changing the interval affects the frequency of stuttering).  I'm also having a hard time differentiating the quality of video with different encoding strategies (general/smart/AI).

Basically I'd just love to see what settings others are using to compare.  I'd especially like to hear from @blakeblackshear if he has time since this model is one of the recommended cameras and I believe he has been using them for a while himself.  If I've overlooked a post already covering this topic then I apologize and would love to get pointed in the right direction.

Thanks!

**Comment by NickM-27** [comment]
I have found https://www.youtube.com/watch?v=BVg81haX2c0&t=718s really helpful for exposure / lighting settings

You can also find the different stream setups in the docs https://docs.frigate.video/guides/camera_setup#example-camera-configuration

**Comment by mdisabato** [comment]
I have Lorex E841CD cameras. They were owned by Dahua until late 2022, and I can use the SmartPSS utility to configure them past what the internal camera web UI allows. I am also running Scrypted to get the cams into HomeKit, but that may change. Scrypted recommends I-frame rates of 4x the frame rate. (ex, If frame rate = 30 fps, I-frame = 120.) That seems to work ok for Scrypted and Frigate. If you look at the attached screen cap, you can see the difference between the two streams. Hope this helps.

![Screenshot 2023-04-09 at 13 10 04](https://user-images.githubusercontent.com/22553313/230789387-cfb0ceca-c169-424d-a6f3-813ef7ef3ca2.png)
